Org Browser + Multi Select Files (#3639)
forcedotcom/salesforcedx-vscode: docs/_articles/en/user-guide/development-models.md

![Deploy source to org](./images/deploy-source-to-org.png)

You can deploy source of a manifest, source files, directories, or folders:

- Manifest
  - In VS Code explorer or editor, right-click a manifest file and select **SFDX: Deploy Source in Manifest in Org**.
  - With a manifest file open in the editor, open the Command Palette and run **SFDX: Deploy Source in Manifest in Org**.
- Source File or Directory

> - In VS Code explorer, right-click single or multi-selected source files or directories and select **SFDX: Deploy Source to Org**.

Reviewer: How does the user select multiple files, is it through right-click?

Author: the standard explorer way -- shift select.

> - In VS Code explorer, right-click single or multi-selected source files or directories and select **SFDX: Deploy Source to Org**.

Reviewer: ```suggestion
- In Org Browser explorer, right-click a single file or directory, or select multiple source files or directories, then select **SFDX: Deploy Source to Org**.
```

### Retrieve Source

You can retrieve a component or multiple components to your local project from the default org:

> - In the org browser, click the retrieve button next to the component or the metadata type.

Reviewer: ```suggestion
- In Org Browser, click the retrieve button next to the component or the metadata type.
```
